euphoria-flink/src/main/java/cz/seznam/euphoria/flink/batch/io/DataSourceWrapper.java

```diff
@@ -43,14 +43,19 @@
   /** currently opened reader (if any) */
   private transient BoundedReader<T> reader;
 
+  private final List<BoundedDataSource<T>> splits;
+
   public DataSourceWrapper(
       BoundedDataSource<T> dataSource,
       BiFunction<LocatableInputSplit[], Integer,
-      InputSplitAssigner> splitAssignerFactory) {
-    
+          InputSplitAssigner> splitAssignerFactory,
```

Comment:
formatting

Reply:
This is intentional and correct, because there is continuation indent.